index.html

```diff
@@ -220,6 +219,95 @@ <h3>Skills</h3>
 			</div>
 		</section>
 
-		<script src="main.js"></script>
-	</body>
-</html>
+        <section id="contact">
```

Comment:
- [ ] As mentioned above, please make sure that your design is fully responsive. Your design is breaking at `100%` zoom ratio. However, it is working fine at `75%` zoom ratio and below. Please fix this. I know you can do that 💪
- [ ] Also, please make sure that your `margin` `padding` is exactly according to figma template


## Your design 👇

https://user-images.githubusercontent.com/106119105/213691318-032329ad-30bd-4673-80d6-d34a227abc02.mp4

## Figma design 👇
![image](https://user-images.githubusercontent.com/106119105/213691393-8abc7563-f36c-444b-b1b5-03073a790a5e.png)

Reply:
- [ ]  You have done great fixing some of the issues with the contact form. You still have a little more work to do to complete the project 😄  Please fix the overlapping form and projects. Thia happens as you change from mobile version to desktop

_**Your design**_
![break](https://user-images.githubusercontent.com/99656687/213873213-c7b75127-aafa-4b0e-9327-b54069bf37f7.png)